src/Microsoft.Health.Fhir.Shared.Core/Features/Operations/Export/GroupMemberExtractor.cs

```diff
@@ -106,9 +106,9 @@ private async Task<HashSet<string>> GetGroupPatientIdsHelper(string groupId, Dat
                 if (
                     (member.Inactive == null
                     || member.Inactive == false
-                    || member.Period?.EndElement == null
-                    || member.Period?.EndElement > fhirGroupMembershipTime
                     || includeInactiveMembers)
```

Comment:
Also, do we need the `includeInactiveMembers` flag? Is that used anywhere?

Reply:
No, I should remove it. I added it to give us the option in the future, but there is no planned use case for it.